contracts/mainnet/connectors/swap-aggregator-lite/helpers.sol

```diff
@@ -0,0 +1,71 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import {InstaConnectors} from "../../common/interfaces.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+
+contract SwapHelpers {
+    struct SwapAmounts {
+        uint256 sellAmount;
+        uint256 buyAmount;
+    }
+
+    struct SwapResult {
+        bool success;
+        bytes returnData;
+        string connector;
+    }
+
+    /**
+     * @dev Instadapp Connectors Registry
+     */
+    InstaConnectors internal constant instaConnectors =
+        InstaConnectors(0x97b0B3A8bDeFE8cB9563a3c610019Ad10DB8aD11);
+
+    /**
+     * @dev Swap using the dex aggregators.
+     * @param connectors_ name of the connectors in preference order.
+     * @param datas_ data for the swap cast.
+     */
+    function _swap(
+        string[] memory connectors_,
+        bytes[] memory datas_
+    )
+        internal
+        returns (SwapResult memory swapResult_)
+    {
+        uint256 length_ = connectors_.length;
+        require(length_ > 0, "zero-length-not-allowed");
+        require(datas_.length == length_, "calldata-length-invalid");
+
+        (bool isOk, address[] memory connectors) = instaConnectors.isConnectors(
```

Comment:
nit: consider using a name like "connectorAddresses" here so there can be no confusion in reading code 

connectors_ vs connectors